src/fragments/lib/storage/js/download.mdx

```diff
@@ -137,11 +139,19 @@ You can use `expires` option to limit the availability of your URLs. This config
 await Storage.get('filename.txt', { expires: 60 });
 ```
 
+### Check existence of file
```

Comment:
Nit: Arguable if this is better but it feels a bit more like treating the existence as a noun?
```suggestion
### Check for existence of a file
```

Reply:
Thanks for the suggestion.